frameit/lib/frameit/config_parser.rb

```diff
@@ -87,9 +88,12 @@ def validate_values(values)
             UI.user_error! "show_complete_frame must be a Boolean" unless [true, false].include?(value)
           when 'font_scale_factor'
             UI.user_error!("font_scale_factor must be numeric") unless value.kind_of?(Numeric)
+          when 'font_height_factor'
+            UI.user_error!("font_height_factor must be numeric") unless value.kind_of?(Numeric)
           end
         end
       end
     end
+    # rubocop:enable Metrics/PerceivedComplexity
```

Comment:
@ohayon I don't think so adding the additional switch case increased the complexity to 19.